packages/release/preconfigured.target

```diff
@@ -0,0 +1,8 @@
+[Unit]
+Description=Bottlerocket dynamic pre-configuration complete
+After=basic.target
+Requires=basic.target
+AllowIsolate=yes
```

Comment:
We might also want:
```
RefuseManualStart=yes
RefuseManualStop=yes
```

For both targets, if it doesn't interfere with isolate.